tests/system/action/user/test_create.py

```diff
@@ -361,123 +585,95 @@ def test_create_permission_manage_user(self) -> None:
-                "vote_delegated_$_to_id": {"1": self.user_id},
-                "vote_delegations_$_from_ids": {"4": [3, 4]},
-                "group_$_ids": {"1": [2, 3], "4": [5]},
+                "username": "usersname",
+                "committee_$_management_level": {
+                    "60": CommitteeManagementLevel.CAN_MANAGE,
+                    "63": CommitteeManagementLevel.CAN_MANAGE,
+                },
+                "committee_ids": [60],
             },
         )
         self.assert_status_code(response, 200)
         self.assert_model_exists(
-            "user/5",
+            "user/3",
             {
-                "username": "username",
-                "number_$": ["1", "4"],
-                "number_$1": "number1",
-                "number_$4": "number1 in 4",
-                "structure_level_$": ["1"],
-                "structure_level_$1": "structure_level 1",
-                "vote_weight_$": ["1"],
-                "vote_weight_$1": "12.002345",
-                "about_me_$": ["1"],
-                "about_me_$1": "about me 1",
-                "comment_$": ["1"],
-                "comment_$1": "comment zu meeting/1",
-                "vote_delegated_$_to_id": ["1"],
-                "vote_delegated_$1_to_id": self.user_id,
-                "vote_delegations_$_from_ids": ["4"],
-                "vote_delegations_$4_from_ids": [3, 4],
-                "group_$1_ids": [2, 3],
-                "group_$4_ids": [5],
+                "committee_ids": [60],
+                "committee_$60_management_level": CommitteeManagementLevel.CAN_MANAGE,
+                "committee_$63_management_level": CommitteeManagementLevel.CAN_MANAGE,
+                "username": "usersname",
             },
         )
 
-    def test_create_permission_user_can_manage_no_permission(self) -> None:
-        """ May create group A, B and C fields """
+    def test_create_permission_group_D_no_permission(self) -> None:
+        """ May not create Group D committee fields, because of missing CML permission for one committee """
         self.permission_setup()
         self.create_meeting(base=4)
-        self.set_organisation_management_level(
-            OrganisationManagementLevel.CAN_MANAGE_USERS, self.user_id
+        self.set_models(
+            {
+                f"user/{self.user_id}": {
+                    "committee_$60_management_level": CommitteeManagementLevel.CAN_MANAGE,
+                    "committee_$_management_level": ["60"],
+                },
+            }
         )
-        self.set_user_groups(self.user_id, [3, 6])
-        self.set_group_permissions(3, [Permissions.User.CAN_MANAGE])
 
         response = self.request(
             "user.create",
             {
-                "username": "username",
-                "number_$": {"1": "number1", "4": "number1 in 4"},
+                "username": "usersname",
+                "committee_$_management_level": {
```

Comment:
Same here. I would expect, that `committee_ids` must also be present.

Reply:
done